src/main/java/com/minecolonies/coremod/client/render/worldevent/RenderTypes.java

```diff
@@ -24,9 +24,21 @@ public static RenderType worldEntityIcon(final ResourceLocation resLoc)
         return InnerRenderTypes.WORLD_ENTITY_ICON.apply(resLoc);
     }
 
+    /**
+     * Used to draw overlay lines that only appear outside existing blocks.
+     */
+    public static RenderType LINES_OUTSIDE_BLOCKS = InnerRenderTypes.LINES_OUTSIDE_BLOCKS;
+
+    /**
+     * Used to draw overlay lines that only appear inside existing blocks.
+     */
+    public static RenderType LINES_INSIDE_BLOCKS = InnerRenderTypes.LINES_INSIDE_BLOCKS;
+
     public static final class InnerRenderTypes extends RenderType
     {
         private static final DepthTestStateShard ALWAYS_DEPTH_TEST = new AlwaysDepthTestStateShard();
+        private static final DepthTestStateShard LEQUAL_DEPTH_TEST = new DepthTestStateShard("depth_lequal", GL11.GL_LEQUAL);
```

Comment:
There are vanilla shards for at least one of them

Reply:
Changed.